### PR TITLE
Basic speed search for the Inspector Tree.

### PR DESCRIPTION
**Pull request by pq**
Fixes: #1697

image

This gets us simple search using Widget `toString()`s.  The Preview View takes it further (supporting path searches and more).  If we want to do that, we should consider adding something like a `getSpeedSearch()` method to `DiagnosticNode`.  Happy to follow-up some refinements if there's interest!

@jacob314 @scheglov

### CODE REVIEW

**Comment by scheglov** [comment]
LGTM

BTW, I did not get highlighting matching substrings without special code in our `ColoredTreeCellRenderer` implementation. Does it work for you in the inspector?

**Comment by pq** [comment]
@scheglov : good catch.  TBH I didn't even notice the rendering.  Looks much better with it, so thanks!

**Comment by pq** [comment]
EDIT: the rendering bits aren't quite ready to land...  I need to make a few tweaks.

File: src/io/flutter/view/InspectorPanel.java [hunk 1]
```diff
@@ -782,6 +804,10 @@ public void customizeCellRenderer(
         setIcon(icon);
       }
     }
+
+    private void appendSearch(@NotNull String text, @NotNull SimpleTextAttributes attributes) {
```
Comment (jacob314) [inline src/io/flutter/view/InspectorPanel.java]: nit: maybe just keep calling this append as it is still just appending text but now it is appending text in a way that is compatible with search.

Comment (pq) [reply src/io/flutter/view/InspectorPanel.java]: `append` is final in super but I agree so made it `appendText` 👍

**Pull request merged**